### PR TITLE
Subgraph Fixes + Refactor

### PR DESCRIPTION
**Pull request by 0xdavinchee**
* Governance Contract will be a template data source, so we no longer hardcode the governance address here
* Reduce number of balance of RPC queries where appropriate to use manual accounting

### CODE REVIEW

File: packages/subgraph/src/mappingHelpers.ts [hunk 1]
```diff
@@ -523,19 +523,31 @@ export function updateAggregateIDASubscriptionsData(
...
+    if (balanceDelta && accountTokenSnapshot.totalSubscriptionsWithUnits == 0) {
         accountTokenSnapshot.balanceUntilUpdatedAt =
-            newBalanceResult.value.value0;
+            accountTokenSnapshot.balanceUntilUpdatedAt.plus(balanceDelta as BigInt);
+    } else {
+        // if the account has any subscriptions with units we assume that
+        // the balance data requires a RPC call for balance because we did not
+        // have claim events there and we do not count distributions
+        // for subscribers
+        let newBalanceResult = superTokenContract.try_realtimeBalanceOf(
```
Comment (hellwolf) [inline packages/subgraph/src/mappingHelpers.ts:541]: piggybacked changes? we had issue with this?

**Comment by hellwolf** [comment]
@0xdavinchee do you want to test it in a feature endpoint?

File: packages/subgraph/src/mappingHelpers.ts (hunk 1 above)
Comment (0xdavinchee) [reply packages/subgraph/src/mappingHelpers.ts:541]: yeah, the balanceOf stuff is piggybacked changes - there were no issues, but this should speed up indexing.

**Comment by 0xdavinchee** [comment]
> @0xdavinchee do you want to test it in a feature endpoint?

yeah, I will deploy to goerli and matic feature

**Review by hellwolf — APPROVED** [review]

**Pull request merged**